### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by roncain**

### CODE REVIEW

**Comment by roncain** [comment]
cc: @richlander @martinwoodward @iamjasonp for feedback

File: README.md [hunk 1]
```diff
@@ -51,7 +57,7 @@ The repo contains the following library components.
 
 ## License
 
-Windows Communication Foundation (including the wcf repo) is licensed under the [MIT license](LICENSE).
+This WCF repo is licensed under the [MIT license](LICENSE).
```
Comment (martinwoodward) [inline README.md]: project

File: README.md [hunk 2]
```diff
@@ -65,3 +71,11 @@ There are many .NET related projects on GitHub.
...
 - [dotnet.github.io](http://dotnet.github.io) is a good place to discover .NET Foundation projects.
+
+## The Windows Communication Foundation Story
+
+[Windows Communication Foundation](https://msdn.microsoft.com/en-us/library/dd456779.aspx) has been part of the full .NET Framework for a number of years. Microsoft continues to develop this version and has made the source code available via [Reference Source](http://referencesource.microsoft.com).
+
+With the introduction of Windows 8, Microsoft made some of the client-oriented functionality available in the [Windows Store](https://msdn.microsoft.com/en-us/library/hh556233(v=vs.110).aspx).  This allowed mobile devices to interact with WCF services that were built with the .NET framework version.
+
+This new WCF GitHub repository differs from these other products in 2 ways. First, it is built on .NET Core so that it can be used on a wider range of devices and operating system. Second, it is  open-source and managed by the [.NET Foundation](http://www.dotnetfoundation.org/projects). Microsoft builds  and publishes NuGet packages from the code in this repository.  UWP and ASP.NET 5 applications use those packages.
```
Comment (martinwoodward) [inline README.md]: This new WCF project differs...

File: README.md [hunk 3]
```diff
@@ -65,3 +71,11 @@ There are many .NET related projects on GitHub.
 - The [.NET Core repo](https://github.com/dotnet/core) links to .NET Core related projects from Microsoft.
 - The [ASP.NET home repo](https://github.com/aspnet/home) is the best place to start learning about ASP.NET 5.
 - [dotnet.github.io](http://dotnet.github.io) is a good place to discover .NET Foundation projects.
+
+## The Windows Communication Foundation Story
```
Comment (martinwoodward) [inline README.md]: I like this section

**Comment by martinwoodward** [comment]
Overall, love these changes.  Some minor nits but otherwise +1

File: README.md [hunk 4]
```diff
@@ -8,11 +8,13 @@
...
+This repository contains the [.NET Core](http://github.com/dotnet/core) version of the [Windows Communication Foundation](https://msdn.microsoft.com/en-us/library/dd456779.aspx) client libraries.
 
-The version of WCF in this repo runs on top of the [.NET Core](http://github.com/dotnet/core) platform, namely the [.NET Core Runtime (CoreCLR)](https://github.com/dotnet/coreclr) and [.NET Core Framework (CoreFx)](http://github.com/dotnet/corefx)
+It's a subset of the .NET Framework version of Windows Communication Foundation and currently supports the same functionality available for Windows 8.1 Store apps. It is used to build .NET Core apps, including [Windows UWP](https://msdn.microsoft.com/en-us/library/windows/apps/dn609832.aspx) and [ASP.NET 5](http://weblogs.asp.net/scottgu/introducing-asp-net-5). These client libraries are suitable for mobile devices or on mid-tier servers to communicate with existing WCF services.
```
Comment (iamjasonp) [inline README.md]: Consider "currently supports the same functionality..." -> "will support the same functionality" or "currently implements the same contract... " ? 

I'm at a loss for words right now, but I want to say that we currently support the same *contract* as Windows Store 8.1 apps but the *functionality* is still work in progress.

**Comment by iamjasonp** [comment]
:+1: - really like the new updates.

**Comment by iamjasonp** [comment]
Consider updating dotnet/home#23 with "WCF Client Libraries" once we have our name at the top of the README finalized.

**Comment by roncain** [comment]
Thanks for the suggestions.  I've incorporated them in this update to the PR.

File: README.md [hunk 5]
```diff
@@ -1,18 +1,20 @@
...
+This repository contains the [.NET Core](http://github.com/dotnet/core) version of the [Windows Communication Foundation](https://msdn.microsoft.com/en-us/library/dd456779.aspx) client libraries.
 
-The version of WCF in this repo runs on top of the [.NET Core](http://github.com/dotnet/core) platform, namely the [.NET Core Runtime (CoreCLR)](https://github.com/dotnet/coreclr) and [.NET Core Framework (CoreFx)](http://github.com/dotnet/corefx)
+It's a subset of the .NET Framework version of Windows Communication Foundation and currently supports the same API surface available for Windows 8.1 Store apps. It is used to build .NET Core apps, including [Windows UWP](https://msdn.microsoft.com/en-us/library/windows/apps/dn609832.aspx) and [ASP.NET 5](http://weblogs.asp.net/scottgu/introducing-asp-net-5). These client libraries are suitable for mobile devices or on mid-tier servers to communicate with existing WCF services.
```
Comment (iamjasonp) [inline README.md]: :+1: - "the same API surface" sounds great!

**Comment by iamjasonp** [comment]
ACK. We are good to merge.

**Pull request merged**